purchase_order_line_schedule/wizards/schedule_order_line_views.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="schedule_order_line_form" model="ir.ui.view">
+        <field name="name">schedule.order.line.form</field>
+        <field name="model">schedule.order.line</field>
+        <field name="arch" type="xml">
+            <form string="Schedule Lines">
+                <group>
+                    <field name="order_line_id" readonly="1" />
```

Comment:
Can we also show ordered quantity here? Makes it easier to spread the quantities.

Reply:
Feedback still valid in my opinion.